docs/manual/developer/03_creating_content.md

```diff
@@ -1301,3 +1301,229 @@ Each rule in the benchmark in the `/linux_os/guide` directory must be a member o
+Expansion of a rule allows it to handle the divergence, whereas spawning creates a new rule according to the new behavior, resulting in two separate single-purpose rules.
+Each approach has its benefits and drawbacks that need to be evaluated, and they are discussed in the next section.
+
+
+#### Adding conditionals to a rule
+
+One way to handle divergences in a rule is to add conditionals to it.
+The conditionals need to be explained in the rule description and added to the checks and remediations, so that each divergence is identified and checked correctly.
+In general, the remediation process should be capable of identifying all compliant states, ensuring idempotence, while also favoring a specific approach in cases where the system is incompliant.
+
+If the divergence is cross-product, the conditionals can be handled at build-time through the techniques mentioned previously.
+An in-product divergence will require the use of conditionals that can be evaluated at scan-time.
+
+#### Spawning a new rule
+
+Another way to handle divergences is to create rules that will handle the newly required behavior.
+Each rule will describe what they check for and remediate, they will be pretty similar but different regarding their specific divergence.
+
+If the divergence is cross-product, the rule only needs to have the appropriate prototype.
+If the divergence is in-product, the rules will need to have disjoint applicabilities in their platforms.
+
+### Aspects to consider when picking one approach
+
+There is no direct guidance on how to handle every case of divergence.
+Evaluate the following aspects against the rule update you are dealing, and pick your poison.
+
+#### Granularity and reusability
+
+##### Conditionals
+
+Pros
+
+- Expected behavior of the rule is maintained, no expectation is violated.
+- Rules may already be expected to handle different conditions, now it’s only one more such condition.
+- Lower granularity means less rules in the project without compromising on the abilities of the project.
+
+Cons
+
+- Profiles or policies that allow or require only one of the behaviors don’t align well with polymorphic rules.
+
+##### Spawning
+
+Pros
+
+- The original rule and the spawned one are clear and direct about what they do.
+- The increased rule granularity promotes reusability, making such rules great  blocks for profiles and controls.
+
+Cons
+
+- Too much granularity in profiles and controls may create noise and impact the user experience.
+- Spawning may result in creation of almost identical rules without any real benefit of granularity.
+- Bugs affecting multiple spawned rules may be more laborious to be consistently fixed along the time.
+
+Profile and control selections
+
+##### Conditionals
+
+Pros
+
+- Unlike with spawning, no changes in profile selections, controls or tailoring.
+
+
+##### Spawning
+
+Remarks
+
+- The ability to define extendable controls may mitigate the shortcomings mentioned right above.
+
+Content Clarity
```

Comment:
I would suggest making these pseudo headings bold, to help communicate these are heading like objects.